Corona-Warn-App/src/test/java/de/rki/coronawarnapp/risk/changedetection/CombinedRiskLevelChangeDetectorTest.kt

```diff
@@ -288,7 +288,7 @@ class CombinedRiskLevelChangeDetectorTest : BaseTest() {
                 .toTypedArray()
         )
 
-        every { riskLevelSettings.lastChangeCheckedRiskLevelCombinedTimestamp } returns Instant.parse("2022-01-03")
+        every { riskLevelSettings.lastChangeCheckedRiskLevelCombinedTimestamp } returns flowOf(Instant.parse("2022-01-03"))
```

Comment:
Please use a proper instant format here, I know it was like that , but considering Java time migration "2022-01-03" does not parse properly as Instant in java

Reply:
Changed all string dates for this test